src/index.js

```diff
@@ -180,6 +194,25 @@ function bookRequestedTrip() {
     // console.log(requestedTrip)
 }
 
+function validateLogIn() {
+    let loginArea = document.querySelector('.login');
+    if (checkUserName() && loginPassword.value === 'travel2020') {
+        fetchStuff(checkUserName());
+        loginArea.classList.add('hidden');
+        requestsButton.classList.remove('hidden');
+        viewTrips.classList.remove('hidden');
+    }
+}
+
+function checkUserName() {
+    if(loginUserName.value.split('traveler')[1]) {
+        let userID = loginUserName.value.split('traveler')[1];
```

Comment:
This is a super clean implementation

Reply:
Thanks! I know this might get a little funky if somebody puts something like `travelerGraveler`, but i think because `Graveler` is a pokemon, and not a valid userID, the find method wont return anything any ways.